targets/targets.json

```diff
@@ -1782,6 +1782,18 @@
         ],
         "device_name": "STM32F413ZH"
     },
+    "MCU_STM415xG": {
+      "inherits": [
+        "MCU_STM32F4"
+      ],
+      "public": false,
+      "extra_labels_add": [
+        "STM32F415xG"
+      ],
+      "macros_add": [
+        "STM32F415xG"
```

Comment:
Correct macro is STM32F415xx
(see in targets/TARGET_STM/TARGET_STM32F4/STM32Cube_FW/CMSIS/stm32f4xx.h)